packages/core/src/browser.js

```diff
@@ -124,68 +166,37 @@ export default class Browser extends EventEmitter {
       }));
     }
 
-    // trigger rejecting pending page callbacks
-    for (let page of this.pages.values()) {
-      page._handleClose();
+    // trigger rejecting pending session callbacks
+    for (let session of this.sessions.values()) {
+      session._handleClose();
     }
 
-    // clear callback and page references
+    // clear own callbacks and sessions
     this.#callbacks.clear();
-    this.pages.clear();
-
-    // resolves when the browser has closed
-    let closed = new Promise(resolve => {
-      /* istanbul ignore next: race condition paranoia */
-      if (!this.process || this.process.exitCode) resolve();
-      else this.process.on('exit', resolve);
-    });
+    this.sessions.clear();
 
-    // force close if needed and able to
-    let kill = () => {
-      /* istanbul ignore next:
-       *   difficult to test failure here without mocking private properties */
-      if (this.process?.pid && !this.process.killed) {
-        try { this.process.kill('SIGKILL'); } catch (error) {
-          throw new Error(`Unable to close the browser: ${error.stack}`);
-        }
+    /* istanbul ignore next:
+     *   difficult to test failure here without mocking private properties */
+    if (this.process?.pid && !this.process.killed) {
+      // always force close the browser process
+      try { this.process.kill('SIGKILL'); } catch (error) {
+        throw new Error(`Unable to close the browser: ${error.stack}`);
       }
-    };
+    }
 
-    /* istanbul ignore else:
-     *   difficult to test failure here without mocking private properties */
-    if (this.profile) kill();
-    else this.send('Browser.close').catch(() => kill());
+    // close the socket connection
+    this.ws?.close();
 
-    // after closing, attempt to clean up the profile directory
-    await closed.then(() => new Promise(resolve => {
-      // needed due to a bug in Node 12 - https://github.com/nodejs/node/issues/27097
-      this.process?.stdin.end();
-      this.process?.stdout.end();
-      this.process?.stderr.end();
-
-      /* istanbul ignore else: sanity */
-      if (this.profile) {
-        rimraf(this.profile, error => {
-          /* istanbul ignore next:
-           *   this might happen on some systems but ultimately it is a temp file */
-          if (error) {
-            this.log.debug('Could not clean up temporary browser profile directory.');
-            this.log.debug(error);
-          }
-
-          resolve();
-        });
-      } else {
-        resolve();
-      }
-    }));
+    // wait for the browser to close
+    return this._closed;
   }
 
-  async page(options) {
-    // create and attach to a new page target returning the resulting page instance
-    let { targetId } = await this.send('Target.createTarget', { url: 'about:blank' });
+  async page(options = {}) {
+    let { targetId } = await this.send('Target.createTarget', { url: '' });
```

Comment:
I swear the about:blank page trolled windows on Chrome 91 (before we opted to go 92)

Reply:
`about:blank` still loads when passing an empty URL. This was left over from me testing that theory and never cleaned up since it doesn't actually change anything